### DISCUSSION TITLE
Usage with 'rasterize-vector-tiles'

### DISCUSSION THREAD

**Discussion by JaffaKetchup** [onthegomap/planetiler#192] [Q&A] [answered]
Hi there,
I would like to use [rasterize-vector-tiles](https://github.com/bjornreppen/rasterize-vector-tiles) to convert the outputted .mbtiles to a .png-based structure.
However, I seem to get stuck using it, and get the same error as in this issue: https://github.com/bjornreppen/rasterize-vector-tiles/issues/6.
Would anyone happen to know what value I need to supply here? I need to know the name of the property which says what each feature's color is.
Many thanks, and happy to provide as much information as I can :)

**Comment by msbarry** [comment, marked as answer]
A full-fledged rasterizing solution like https://github.com/maptiler/tileserver-gl will take the vector tiles and apply a configurable style to them to describe how each feature should be rendered (color, width, etc). It looks like rasterize-vector-tiles is a very simple approach that doesn't use an external style, and instead requires the vector tile features to have a color embedded on them (in the attribute you pass in as colorprop). So your 2 options for that tool would be:
- customize planetiler code to add a color attribute to each feature
- or customize rasterize-vector-tile to infer a color for each feature (ie. based on the layer name)

**Reply by JaffaKetchup** [reply]
Ah, thanks for the quick answer. I guess I'll have to keep searching for rasterizers then, as I need to get the full set of .pngs out, and editing the projects isn't really an option for me in this case (creating some updated documentation for another map library).
Many thanks anyway :)
